R/compute_long_short_returns.R

```diff
@@ -0,0 +1,27 @@
+#' Compute long-short portfolio returns
+#'
+#' Compute long-short portfolio returns
+#'
+#' @param portfolio_returns Some dataset...
+#'
+#' @return A tibble with long-short returns.
+#'
+#' @examples
+#' \donttest{
+#'   compute_long_short_returns(portfolio_returns)
+#' }
+#'
+#' @export
+compute_long_short_returns <- function(
+    portfolio_returns,
+    direction = "top_minus_bottom"
+) {
+  portfolio_returns |>
+    filter(portfolio %in% c(1, max(portfolio))) |>
```

Comment:
min(portfolio) instead of 1 seems logical to me.

Reply:
(unless there is a guarantee that assign_portfolio will always return a portfolio 1, which I doubt in all generality).